.github/workflows/ci.yaml

```
@@ -42,6 +40,7 @@ jobs:
             git \
             libelf-dev \
             libtool \
+            linux-tools-`uname -r` \
```

nit: make shellcheck happy 😉 

```suggestion
            linux-tools-$(uname -r) \
```